syntax = "proto3";

option csharp_namespace = "IntelliCook.IngredientRecognition.Client";

message RecognizeIngredientsStreamRequest {
    bytes image = 1;
}

message RecognizeIngredientsResponse {
    repeated RecognizeIngredientsIngredient ingredients = 1;
}

message RecognizeIngredientsIngredient {
    string name = 1;
    float x = 2;
    float y = 3;
    float width = 4;
    float height = 5;
}
